src/app/forms/custom-sections/weights/weights.component.ts

```diff
@@ -103,20 +103,23 @@ export class WeightsComponent implements OnInit, OnDestroy, OnChanges {
   }
 
   private handleVehicleTechRecordChange(changes: SimpleChanges): void {
-    if (changes['vehicleTechRecord'] && !this.ladenWeightOverride && this.form) {
-      const { currentValue, previousValue } = changes['vehicleTechRecord'];
+    const { vehicleTechRecord } = changes;
+    if (this.form && vehicleTechRecord && !this.ladenWeightOverride) {
+      const { currentValue, previousValue } = vehicleTechRecord;
 
       const fieldsChanged = [
         'techRecord_seatsUpperDeck',
         'techRecord_seatsLowerDeck',
         'techRecord_manufactureYear',
-        'techRecord_grossKerbWeight',
-        'techRecord_standingCapacity'
+        'techRecord_grossKerbWeight'
       ].some(field => currentValue[field] !== previousValue[field]);
 
-      if (fieldsChanged && currentValue.techRecord_manufactureYear) {
-        this.form.patchValue({ techRecord_grossLadenWeight: this.calculateGrossLadenWeight() }, { emitEvent: false });
+      if (fieldsChanged && this.vehicleTechRecord.techRecord_vehicleType === 'psv') {
```

Comment:
This if is missing the manufacture year flag, we only want to auto calculate the year of manufacture when there is a manufacture year present